account_financial_report/readme/DESCRIPTION.rst

```diff
@@ -14,3 +14,5 @@ currency used in account move lines is properly shown.
 
 In case that in an account has not been configured a second currency foreign
 currency balances are not available.
+
+Invoicing / Settings / Invoicing / OCA Aged Report Configuration you will be able to set dynamic intervals that will appear on the Aged Partner Balance
```

Comment:
This should go better redacted on CONFIGURE.rst. And please respect 80 cols limit.